src/Parser/Monk/Windwalker/Modules/Spells/StormEarthAndFire.js

```diff
@@ -36,6 +46,57 @@ class StormEarthAndFire extends Analyzer{
     const reducedCooldownWithTraits = 90 - this.traitsCDReduction;
     return reducedCooldownWithTraits;
   }
+
+  on_toPlayer_applybuff(event) {
+    const spellId = event.ability.guid;
+    if (SPELLS.STORM_EARTH_AND_FIRE_CAST.id === spellId || SPELLS.SERENITY_TALENT.id === spellId) {
+      this.sefCount += 1;
+    }
+  }
+
+  on_byPlayer_cast(event) {
+    const spellId = event.ability.guid;
+
+    if (!this.combatants.selected.hasBuff(SPELLS.STORM_EARTH_AND_FIRE_CAST.id) && !this.combatants.selected.hasBuff(SPELLS.SERENITY_TALENT.id)) {
+      return;
+    }
+    if (SPELLS.RISING_SUN_KICK.id === spellId) {
+      this.risingSunKicks += 1;
+    }
+    if (SPELLS.FISTS_OF_FURY_CAST.id === spellId) {
+      this.fistsOfFuries += 1;
+    }
+    if (SPELLS.STRIKE_OF_THE_WINDLORD.id === spellId) {
+      this.strikeOfTheWindlords += 1;
+    }
+    if (SPELLS.WHIRLING_DRAGON_PUNCH_TALENT.id === spellId) {
+      this.whirlingDragonPunches += 1;
+    }
+  }
+
+
+  statistic() {
+   const icon = this.combatants.selected.hasTalent(SPELLS.SERENITY_TALENT.id) ? SPELLS.SERENITY_TALENT : SPELLS.STORM_EARTH_AND_FIRE_CAST;
+   return (
+     <StatisticBox
+       icon={<SpellIcon id={icon.id} />}
+       label={
+          <div>
+          During your {this.sefCount} {icon.name}s you cast: 
+          <li>{this.risingSunKicks}/{this.sefCount*2+1} Rising Sun Kicks</li> 
+          <li>{this.fistsOfFuries}/{this.sefCount} Fists of Furies</li>
+          <li>{this.strikeOfTheWindlords}/{this.sefCount} Strikes of the Windlord</li>
+              {this.whirlingDragonPunches > 0
+              ? <li>{this.whirlingDragonPunches}/{this.sefCount} Whirling Dragon Punches</li> : ''
+              }
+        </div>
+       }
```

Comment:
Seems too large for the label of a statistic box. How about rewording that and moving that to the tooltip so the `label` is smaller, or making use of `substatistic`s like here? 
![image](https://user-images.githubusercontent.com/5068584/35356214-0e9b1998-0150-11e8-9bf7-8564f856f8a3.png)

Reply:
I had it as a label with the list as a tooltip, but i didn't like the way it looked in the statistics box as a greyed out text without also having a value. I had been looking around and wanted to make something that looks like the overload statistic for elemental shamans, which is also a quite large box.